ODataVirtualDataSourceDataProviderWorker.ts

```diff
@@ -475,14 +480,7 @@ export class ODataVirtualDataSourceDataProviderWorker extends AsyncVirtualDataSo
 				let propertyNames = [];
 				let propertyTypes = [];
 				for (let i = 0; i < schema.propertyNames.length; i++) {
-					let found = false;
-					for (let j = 0; j < this._schemaIncludedProperties.length; j++) {
-						if (this._schemaIncludedProperties[j] == schema.propertyNames[i]) {
-							found = true;
-							break;
-						}
-					}
-					if (!found) {
+					if (!this._schemaIncludedProperties.has(schema.propertyNames[i])) {
```

Comment:
You need to include everything If the included set is null though

Reply:
The code should do that already if the _schemaIncludedProperties set is null. There's a null check a few lines above this. If there is a null set then it does not alter the schema so all the properties are included. It only filters out the props if there is a valid set.